__tests__/app.test.js

```diff
@@ -154,16 +154,16 @@ describe('3.GET/api/articles/:article_id', ()=>{
                 body: "I find this existence challenging",
                 created_at: '2020-07-09T20:11:00.000Z',
                 votes: 100,
+                comment_count:11
```

Comment:
We don't want to amend our original test, instead we'd want to add a new test! TDD still applies, so we want our tests to be expandable and buildable.